code/modules/surgery/bodyparts/parts.dm

```diff
@@ -361,6 +361,13 @@
 	px_y = 12
 	max_stamina_damage = 50
 	can_be_disabled = TRUE
+	/// We store this here to generate our icon key more easily.
+	var/left_leg_mask_key
+	/// The associated list of all the left leg mask keys associated to their cached left leg masks.
+	/// It's static, so it's shared between all the left legs there is. Be careful.
+	/// Why? Both legs share the same layer for rendering, and since we don't want to do redraws on
+	/// each dir changes, we're doing it with a mask instead, which we cache for efficiency reasons.
+	var/static/list/left_leg_mask_cache = list()
```

Comment:
any reason this can't just be a global list?

Reply:
It could be a global list, yeah, I just didn't really see a point in making it a global list rather than just a static list, since nothing else will access it anyway. Less chances of it being tampered with.

Reply:
GLOB.thing is slower to access than a static var isnt it

Reply:
not in a way that matters for anything being done here